java/core/src/java/org/apache/orc/Writer.java

```diff
@@ -150,6 +162,7 @@ void appendStripe(byte[] stripe, int offset, int length,
    *
    * @return stripe information
    * @throws IOException
+   * @since 1.6.0
```

Comment:
Looks like this is from 1.6.8, could you fix this?

Reply:
Oh. My bad. Thank you!